DataFormats/L1TrackTrigger/src/classes_def.xml

```diff
@@ -51,5 +51,7 @@
   <class name="edm::Wrapper<tt::TTTrackRefMap>"/>
   <class name="TTDTC"/>
   <class name="edm::Wrapper<TTDTC>"/>
+  <class name="std::vector<std::pair<double, double>>"/>
```

Comment:
This dictionary already exists in
https://github.com/cms-sw/cmssw/blob/134545cee975920459f867c13f8c1a870da1da12/DataFormats/StdDictionaries/src/classes_def_vector.xml#L14
Please remove from here

Reply:
done